docs/docsite/rst/scenario_guides/guide_online.rst

```diff
@@ -0,0 +1,29 @@
+*************************
+Using Ansible with Online
+*************************
+
+Introduction
+============
+
+Online is a French hosting company mainly known for providing baremetal servers named Dedibox.
+Check it out: https://www.online.net/en
```

Comment:
Make this a link so users don't need to cut and paste. Syntax is `'text for link <http://URL>'_`, but replace the single quotation marks with backticks.